hibernate-core/src/test/java/org/hibernate/orm/test/mapping/formula/FormulaTests.java

```diff
@@ -114,7 +114,7 @@ public static class Account {
 		@DialectOverride.Formula(dialect = DB2Dialect.class,
 				override = @Formula("varchar_format(rate * 100) || '%'"))
 		@DialectOverride.Formula(dialect = OracleDialect.class,
-				override = @Formula("to_char(rate * 100) || '%'"))
+				override = @Formula("to_char(cast(rate * 100 as number(10,2))) || '%'"))
```

Comment:
What happens without the cast? What is the format?

Reply:
It renders in exponential form i.e. `1.25E+000`